packages/client-search/src/methods/client/removeUserID.ts

```diff
@@ -11,7 +11,7 @@ export const removeUserID = (base: SearchClient) => {
     const mappedRequestOptions = createMappedRequestOptions(requestOptions);
 
     // eslint-disable-next-line functional/immutable-data
-    mappedRequestOptions.headers['X-Algolia-User-ID'] = userID;
+    mappedRequestOptions.queryParameters['X-Algolia-User-ID'] = userID;
```

Comment:
can we get into similar URL too long problems as with secured api key?

Reply:
Yes - but in reality that doesn't happen.

@aseure @chloelbn any thoughts on this remark?